RELEASE_NOTES

```diff
@@ -1,6 +1,14 @@
 This file contains a description of the major changes to the EESSI
 build-and-deploy bot. For more detailed information, please see the git log.
 
+v0.2.0 (4 october 2023)
```

Comment:
Since this is a minor enhancement (bot itself hasn't changed, it's just an extra script which is currently totally disconnected from the bot), I'd use `0.1.1`, but I guess that's debatable (since it's not a bug fix either)

We need a 3rd vote... @bedroge ?

Reply:
We could argue that not cleaning up was a bug???